app/src/main/java/com/eveningoutpost/dexdrip/GcmActivity.java

```diff
@@ -688,8 +790,25 @@ static void pushCalibration2(double bgValue, String uuid, long offset) {
             GcmActivity.sendMessage(myIdentity(), "cal2", json);
         }
     }
+    public static void pushLibreBlock(String libreBlock) {
+        Log.i(TAG, "libreBlock called: " + libreBlock);
+        if (!Home.get_master()) {
+            return;
+        }
+        if (!JoH.ratelimit("libre-allhouse", 5)) {
```

Comment:
Use `pratelimit` for critical rate limiting as `ratelimit` isn't safe on all platforms.

Reply:
Done.